jaxrs-clients/src/main/java/com/palantir/remoting1/jaxrs/FeignJaxRsClientBuilder.java

```diff
@@ -163,7 +163,7 @@ private Decoder createDecoder(ObjectMapper objectMapper) {
                 .randomGenerator(new Random())
                 .state(new ThreadLocalServerClientAndLocalSpanState(
                         getIpAddress(), 0 /** Client TCP port. */, userAgent))
-                .spanCollector(new SlfLoggingSpanCollector("ClientTracer(" + userAgent + ")"))
+                .spanCollector(new SlfLoggingSpanCollector("tracing.client." + userAgent))
```

Comment:
I don't think `tracing.client.com.palantir.product#product-name;1.2.3-1-gabc123` is easier to parse than `ClientTrace(com.palantir.product#product-name;1.2.3-1-gabc123)`?